### DISCUSSION TITLE
How to reorder Playlists (not tracks inside a playlist)

### DISCUSSION THREAD

**Discussion by KenthJ** [JohnnyCrazy/SpotifyAPI-NET#661] [Q&A] [answered]
Hi there

How do I reorder the playlists itself? (not the tracks inside a playlist)

I can only find 'spotify.Playlists.ReorderItems()' which only seems to be used for reordering the tracks inside a playlist

Thanks a lot in advance

**Comment by JohnnyCrazy** [comment, marked as answer]
Can't be done. See https://github.com/spotify/web-api/issues/1031

**Reply by KenthJ** [reply]
Thanks a lot.

Yes, I can see it's a limitation of the original Spotify API.

I wonder why they have made that limitation, in the Spotify Desktop App, you are able to reorder your playlists by dragging them up or down, so it would be nice to do the same via the API
